src/command_spec.rs

```diff
@@ -93,6 +108,7 @@ mod tests {
                         String::from("--option"),
                         String::from("value"),
                     ],
+                    envs: vec![],
```

Comment:
Can we test environment variables here? Or are they not part of the formatted string?

Reply:
Ugh, it does work, but that's probably not what we want, if we ever add any secrets to our environment variables.

Reply:
Only works on Linux, oddly enough. Added the test for documentation.